src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -244,6 +254,30 @@ def setUp(self):
         if self.setup_start_servers:
             self.start_servers()
 
+            if self.nvme_parameter == "nvme":
+                # Format servers
+                format_res = dmg_utils.storage_format(dmg_hostlist)
+                try:
+                    format_str = "Mntpoint:/mnt/daos Status:CTRL_SUCCESS"
+                    general_utils.poll_pattern(
+                        len(self.hostlist_servers), format_res, 300, format_str)
+                except CommandFailure as details:
+                    self.fail(details)
+
+                # Stop servers
+                server_utils.stop_server(hosts=self.hostlist_servers)
+
+                # Clean and prepare server storage
+                server_utils.clean_server(self.hostlist_servers)
+
+                # Run servers as Jenkins user
+                self.start_servers()
+
+                # Storage prep
+                res = dmg_utils.storage_prep(dmg_hostlist, nvme=True)
+                if res.exit_status != 0:
+                    self.fail(res.stderr)
+
```

Comment:
How long does/can all of this formatting and prepping storage take on the size of devices we are expecting to support?

The concern here is that the time allowed in `setUp()` is not unbounded.  I don't remember what the limit is, but it should be investigated to ensure that we are well clear of it in adding all of this processing to `setUp()`.